.pre-commit-config.yaml

```diff
@@ -21,3 +21,10 @@ repos:
     rev: '4.0.1'
     hooks:
       - id: flake8
+
+  - repo: https://github.com/pycqa/pylint
+    rev: 'v2.13.0'
+    hooks:
+      - id: pylint
+        args:
+        - --max-line-length=80
```

Comment:
We make use of black using the default configuration which has a line length of 88 characters. Can this be reflected in the pylint configuration please.